src/vma/main.cpp

```diff
@@ -406,6 +408,10 @@ void print_vma_global_settings()
 		vlog_printf(VLOG_INFO,"OFED Version: %s\n", ofed_version_info);
 	}
 
+	if (vma_git_version_str[sizeof(VMA_GIT_TITLE)-1]) {
```

Comment:
I think it is not actual after https://github.com/Mellanox/libvma/pull/348#discussion_r110959704
but in previous variant `vma_git_version_str[sizeof(VMA_GIT_TITLE)-1]` is always TRUE.